frames/window_main.lua

```diff
@@ -4879,7 +4880,12 @@ function Details:InstanceRefreshRows(instance)
 				row.icone_classe:Show()
 
 				if (start_after_icon) then
-					row.statusbar:SetPoint("topleft", row.icone_classe, "topright")
+					row.statusbar:ClearAllPoints()
+					row.statusbar:SetPoint("left", row.icone_classe, "right", start_after_icon_offset_x, 0)
```

Comment:
Why an offset? This doesn't solve the issue as far as i understand.

Reply:
The offset isn't required to solve the issue. I've added it only because it looks bad to have no space between icon and bar. I've tested the changes and they work.